src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,79 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.dto.LineUpdateRequest;
+import org.hibernate.annotations.DynamicUpdate;
+
+import javax.persistence.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+@DynamicUpdate
+@Entity
+@Table(name = "line")
+public class Line extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true)
+    private String name;
+    @Column(unique = true)
+    private String color;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn
+    private Station upStation;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn
+    private Station downStation;
+    @Column(nullable = false)
+    private Long distance;
+
+    protected Line() {
+    }
+
+    public Line(String name, String color, Station upStation, Station downStation, Long distance) {
+        this.name = name;
+        this.color = color;
+        this.upStation = upStation;
+        this.downStation = downStation;
+        this.distance = distance;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public List<Station> getStations() {
+        return Arrays.asList(this.upStation, this.downStation);
+    }
+
+    public void update(LineUpdateRequest request) {
```

Comment:
도메인이 Request를 바로 알기보다는 값만 받아오면 어떨까요...?
```suggestion
    public void update(String name, String color) {
```
Request 같은 dto의 변경이 도메인의 변경에 영향을 주는 구조는 피하는 경향이 있는 것 같습니다!

Reply:
네. [927c2de](https://github.com/next-step/atdd-subway-admin/pull/696/commits/927c2ded78ac9bb925a8280d80da09bb4116078c) 에서 값만 받아오도록 변경해보았습니다.